go.sum

```diff
@@ -0,0 +1,11 @@
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```

Comment:
Are these all required for production?

Reply:
no. There was some issue with `go mod tidy`. Even though these pkgs are not imported anywhere, it got added in go.sum. Will check this.

Reply:
`go mod tidy` is adding these packages even though it is not imported in ifsc package, which is confusing.

The output for `go mod why github.com/davecgh/go-spew` returns 
```
# github.com/davecgh/go-spew
(main module does not need package github.com/davecgh/go-spew)
```
So, I assume it's safe to remove go-spew pkg.

However, it might have been added to go.sum because `go mod why -m github.com/davecgh/go-spew` shows that testify pkg needs it.
```
# github.com/davecgh/go-spew
github.com/gyanesh-m/ifsc/src/go
github.com/gyanesh-m/ifsc/src/go.test
github.com/stretchr/testify/assert
github.com/davecgh/go-spew/spew
```